Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
-gem "middleman", "~> 3"#, github: "middleman/middleman", branch: "v3-stable"
-gem "middleman-matomo", github: "barthalion/middleman-matomo"
+gem "middleman", "~> 4"#, github: "middleman/middleman", branch: "v3-stable"
+gem 'matomo-middleman'
```

Comment:
This seems to work fine, as far as I can tell without deploying it. Locally everything seems to be added correctly.

But I'm not sure what the changes in @barthalion fork were

Reply:
I don't actually remember at this point.

Reply:
Guess we figured out why :D